### DISCUSSION TITLE
`<selectlist>` (ex `<selectmenu>`)

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#95] [State of HTML Features]
Stylable, customizable dropdown control.

```html
<selectmenu>
	<option>Option 1</option>
	<option>Option 2</option>
	<option>Option 3</option>
</selectmenu>
```

### Specification URL:

https://open-ui.org/components/selectmenu/

### Any other links?

https://web.dev/state-of-css-2022/#customizing-select-elements
https://css-tricks.com/the-selectmenu-element/
https://www.smashingmagazine.com/2023/06/advanced-form-control-styling-selectmenu-anchoring-api/
https://hidde.blog/custom-select-with-selectmenu/
https://microsoftedge.github.io/Demos/selectmenu/

### How stable is this feature?

Future (No implementations yet)

### Type of feature

HTML element

### What focus area does this feature fall into?

Forms

### Details

Fully stylable, customizable dropdown menus

**Comment by mfreed7** [comment]
Quick comment here - [OpenUI decided to rename this to `<selectlist>`](https://github.com/openui/open-ui/issues/773#issuecomment-1664421419). We should make sure to call it that, and **not** `<selectmenu>`.
